src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIZE = 10;
+    private static final ColorSupplier supplier = new ColorSupplier();
+    private static final Random random = new Random();
+    private static final Circle CIRCLE_DEFAULT = new Circle("WHITE", 10);
```

Comment:
It's better to make "WHITE" and 10 as two constants and create an Object of Circle class in getDefaultFigure() method